tests/cloud_health_api_test.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import unittest
+import sys
+
+#
+# Third party libraries
+#
+
+from mock import MagicMock, patch
+
+#
+# Internal libraries
+#
+
+from krux_cloud_health.cloud_health_api import Application, main
+from krux_cloud_health.cloud_health import Interval, NAME
+from krux.stats import DummyStatsClient
+
+
+class CloudHealthAPITest(unittest.TestCase):
+
+    NAME = 'cloud-health-tech'
+    API_KEY = '12345'
+
+    @patch('sys.argv', ['api-key', API_KEY])
+    def setUp(self):
+        self.app = Application()
+        self.app.logger = MagicMock()
+        self.app.cloud_health.cost_history = MagicMock()
+        self.app.cloud_health.cost_history.return_value = {
+            'Total': {'key': 'value'},
+            '2016-05-01': {'key': 'value'},
+            '2016-06-01': {'key': 'value'},
+            '2016-07-01': {'key': 'value'},
+        }
+
+    def test_add_cli_arguments(self):
+        """
+        Cloud Health API Test: All arguments from present in the args
+        """
+        self.assertIn('api_key', self.app.args)
+        self.assertIn('interval', self.app.args)
+        self.assertIn('set_date', self.app.args)
+
+        self.assertEqual(self.API_KEY, self.app.args.api_key)
+        self.assertEqual(Interval.daily.name, self.app.args.interval)
+        self.assertIsNone(self.app.args.set_date)
+
+    def test_run_error(self):
+        self.app.cloud_health.cost_history = MagicMock(side_effect=ValueError('Error message'))
+        with self.assertRaises(SystemExit):
+            self.app.run()
+        self.app.logger.error.assert_called_once_with('Error message')
```

Comment:
Please test the `sys.exit()` was called with exit value 1. Refer to [this answer](http://stackoverflow.com/a/15672165).